<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>destination</class>
 <widget class="QDialog" name="destination">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion des destinations</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>651</height>
      </rect>
     </property>
     <property name="title">
      <string>&quot;''</string>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>301</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255,0);
border-color: rgb(0, 0, 0);
font: 75 italic 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:15px;
font: 87 9pt &quot;Segoe UI Black&quot;;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Lucida Bright&quot;;</string>
       </property>
       <property name="text">
        <string>Destination</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Lucida Bright&quot;;</string>
       </property>
       <property name="text">
        <string>Durée du séjour</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Lucida Bright&quot;;</string>
       </property>
       <property name="text">
        <string>Hébergement</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Lucida Bright&quot;;</string>
       </property>
       <property name="text">
        <string>Code du voyage</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Lucida Bright&quot;;</string>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_hebergement">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_dure">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_code">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prix">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(255, 118, 21);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}

QPushButton:hover {
	background-color: rgb(255, 140, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}
QPushButton:pressed {
	background-color: rgb(255, 179, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}


</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>144</x>
         <y>230</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(255, 118, 21);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}

QPushButton:hover {
	background-color: rgb(255, 140, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}
QPushButton:pressed {
	background-color: rgb(255, 179, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}


</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_dest">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;MV Boli&quot;;
</string>
       </property>
       <property name="text">
        <string>Formulaire</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: rgb(255, 10, 21);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-style: solid;
    border-radius: 14px;
}

QPushButton:hover {
	background-color: rgb(255, 140, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);

}
QPushButton:pressed {
	
	background-color: rgb(255, 179, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);

}</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pb_rechercher">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 9px;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
      <item>
       <property name="text">
        <string>code</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>destination</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>100</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgb(255, 118, 21);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}

QPushButton:hover {
	background-color: rgb(255, 140, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}
QPushButton:pressed {
	background-color: rgb(255, 179, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}


</string>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>380</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgb(255, 118, 21);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}

QPushButton:hover {
	background-color: rgb(255, 140, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}
QPushButton:pressed {
	background-color: rgb(255, 179, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}


</string>
      </property>
      <property name="text">
       <string>QR Code</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>410</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgb(255, 118, 21);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}

QPushButton:hover {
	background-color: rgb(255, 140, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}
QPushButton:pressed {
	background-color: rgb(255, 179, 0);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}


</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>331</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 9pt &quot;Comic Sans MS&quot;;
font: 87 10pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255,0);


</string>
      </property>
      <property name="title">
       <string>Suppression</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Lucida Bright&quot;;
</string>
       </property>
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="supprimer">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="le_code_supp">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">

QPushButton {
background-color: rgb(255, 0, 4);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}

QPushButton:hover {
	background-color: rgb(198, 0, 3);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}
QPushButton:pressed {
	background-color: rgb(107, 0, 1);
font: 75 9pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>-70</x>
        <y>0</y>
        <width>1091</width>
        <height>691</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../005656le55.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tab_voyage">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>541</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="print">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>410</y>
        <width>191</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="qrcode">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>410</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pb_rechercher</zorder>
     <zorder>comboBox</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>pushButton_14</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>groupBox_4</zorder>
     <zorder>tab_voyage</zorder>
     <zorder>print</zorder>
     <zorder>qrcode</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Calendrier</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>971</width>
       <height>621</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>451</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 16pt &quot;Segoe UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Calendrier Des voyages Organisées</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>381</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>991</width>
        <height>621</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../005656le55.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_8</zorder>
     <zorder>label_14</zorder>
     <zorder>calendarWidget</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Statistiques</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>971</width>
       <height>621</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>-60</x>
        <y>0</y>
        <width>1031</width>
        <height>731</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../005656le55.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>501</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Fire Alarm</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>371</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QPushButton" name="offButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>SIGNAL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="fireLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
